generate.sh

```diff
@@ -161,6 +161,15 @@ if [ $? -ne 0 ]; then
 fi
 print_log "Documentation Generated Successfully!\n"
 
+if [ -n "$APIDOCS_NAME" ] && [ -n "$APIDOCS_URL" ]; then
+  if [ "$APIDOCS_NAME" == "swagger" ]; then
+    git clone -q https://github.com/swagger-api/swagger-ui.git swagger >/dev/null 2>&1
```

Comment:
Instead of cloning their master, which can be unstable, get their latest stable release. 

`https://api.github.com/repos/swagger-api/swagger-ui/releases/latest` would give you a `zipball_url` parameter to get the zip of the latest release. Use that please.